### ISSUE TITLE
Either improvement for better linting experience

### ISSUE THREAD

**Issue by LukaszTheProgrammer** [monet/monet.js#105]
Currently Either has two factory methods .Right and .Left which names does not follow some linters convention. 
For instance having a code like:
`const e = Either.Left("error");`
results in the following error in es-lint:
_"A function with a name starting with an uppercase letter should only be used as a constructor  new-cap"_

Is it possible to at least provide ofLeft as a wrapper to .Left because .Right can easily be replaced with Either.of.

Thanks
Lukasz

**Comment by ulfryk** [comment]
@LukaszTheProgrammer - thanks for suggestion :)

Using just `Left` or `Right` does not satisfy your needs ?

```typescript
import { Left } from 'monet';

const e = Left("error");
```

**Comment by LukaszTheProgrammer** [comment]
@ulfryk it does I just wanted the example to be explicit.

BTW.: Thanks to all of you guys who make monet.

**Comment by ulfryk** [comment]
@LukaszTheProgrammer so can we treat issue as fixed ?

No problem - maybe you'd like to help ?

**Comment by LukaszTheProgrammer** [comment]
@ulfryk

**Comment by LukaszTheProgrammer** [comment]
@ulfryk no because even if you use an import like:
`import { Left } from 'monet';`

linter complains.

I can provide a PR for this.

**Comment by char0n** [comment]
Just to join the discussion I had the same problem. I see that it has been fixed for 0.9.x branch. For 0.8.x import aliasing worked for me just fine
```javascript
// ES2016
import { Reader as reader } from 'monet';
import { Left as left } from 'monet';
import { Either, Right as right } from 'monet'; 
// these two are eqvivalent and linter friendly: Either.of(1) <-> right(1)
```
```javascript
// CommonJS
const { Reader: reader } = require('monet');
const { Left: left } = require('monet');
const { Either, Right: right } = require('monet'); 
// these two are eqvivalent and linter friendly: Either.of(1) <-> right(1)
